app/extensions/brave/locales/en-US/preferences.properties

```diff
@@ -333,6 +333,7 @@ defaultBrowser=Brave is your default browser.
 notDefaultBrowser=Brave is not your default browser.
 setAsDefault=Set as default…
 checkDefaultOnStartup=Always check on startup
-useTorrentViewer=Enable Torrent Viewer (requires browser restart)
+useTorrentViewer=Enable Torrent Viewer *
 dashboardSettingsTitle=Dashboard
 dashboardShowImages=Show images
+requireRestart=* Requires browser restart
```

Comment:
I think there's a typo here (which breaks all l10n for that). 

Should be `requiresRestart` instead